robotframework-ls/src/robotframework_ls/impl/collect_keywords.py

```diff
@@ -262,18 +262,22 @@ def _collect_libraries_keywords(
     # Get keywords from libraries
     from robotframework_ls.impl.robot_constants import BUILTIN_LIB
     from robocorp_ls_core.lsp import CompletionItemKind
+    from robotframework_ls.impl import ast_utils
 
     libraries = completion_context.get_imported_libraries()
 
     library_infos = set(
         _LibInfo(
-            completion_context.token_value_resolving_variables(library.name),
+            completion_context.token_value_resolving_variables(
+                ast_utils.get_library_name_from_node(library)
```

Comment:
I think you should check whether `ast_utils.get_library_name_from_node(library)` returns `None` here.

i.e.: While the user is typing, it's possible for him to just write a `Library   Remote`, and I'm not sure the rest of the code deals with a `None` name properly.